src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -149,6 +150,19 @@ describe('sidebar.components.sidebar-content', function() {
     });
   }
 
+  function createSidebarContent(
```

Comment:
I plan on cleaning up these tests but for now I wanted to make sure I had some tests for this.